setup.py

```diff
@@ -53,6 +53,10 @@
     "iso8601",
 ]
 
+if sys.version_info[:2] == (2, 6):
```

Comment:
Isn't this test inverted? I think you want to add it when it's *not* python 2.6.